src/js/common/schemaform/review/ReviewPage.jsx

```diff
@@ -37,7 +37,8 @@ class ReviewPage extends React.Component {
       // to be slower than shallow cloning objects
       viewedPages: new Set(
         getPageKeys(props.route.pageList, props.form.data)
-      )
+      ),
+      showValidationWarning: false
```

Comment:
Another approach to this is to make another submission state (like `validationError` or something), call `this.props.setSubmission`, and add the error message in SubmitButtons (along with the other failure states).